source/Nevermore.IntegrationTests/Model/ProductSubtotalMap.cs

```diff
@@ -0,0 +1,15 @@
+using Nevermore.Mapping;
+
+namespace Nevermore.IntegrationTests.Model
+{
+    public class ProductSubtotalMap : DocumentMap<ProductSubtotal>
```

Comment:
The test using this previous never made a mapping for it, but it was sorta valid at the time since _RawSqlQuery_ method never actually loaded a map... It does now, and that feels valid to me since it's going to return a document type from it.